syntax="proto3";

package gravity_pb;

import "Dir.proto";

message PlatformEntity {
	float posX = 1;
	float posY = 2;
	float posZ = 3;
	Dir dir = 4;
	float slide_offset_x = 5;
	float slide_offset_y = 6;
	fixed32 name = 7;
	float slide_time = 9;
	float launch_speed = 10;
}
